src/main/java/com/autotune/common/target/kubernetes/service/impl/KubernetesServicesImpl.java

```diff
@@ -420,7 +420,7 @@ public Event getEvent(String namespace, String eventName) {
         try {
             event = kubernetesClient.events().inNamespace(namespace).withName(eventName).get();
         } catch (Exception e) {
-            LOGGER.warn("Watcher not added! Only REST API access is enabled.");
```

Comment:
Why is this message being removed?

Reply:
It was mistakenly added in the wrong method. Updated the `addWatcher` method above and removed it from here.